packages/next-server/server/next-server.ts

```diff
@@ -343,32 +359,54 @@ export default class Server {
     return this.sendHTML(req, res, html)
   }
 
-  private async renderToHTMLWithComponents(
-    req: IncomingMessage,
-    res: ServerResponse,
+  private async findPageComponents(
     pathname: string,
     query: ParsedUrlQuery = {},
-    opts: any,
   ) {
-    const serverless = !this.renderOpts.dev && this.nextConfig.target === 'serverless'
+    const serverless =
+      !this.renderOpts.dev && this.nextConfig.target === 'serverless'
```

Comment:
Shouldn't this be the case for `resolveApiRequest` as well? Currently API routes are broken in development.

Reply:
@kevva Can you create an issue for it please ? :pray:

Reply:
@lfades, #7617.